src/React.Owin/JsxFileMiddleware.cs

```diff
@@ -0,0 +1,71 @@
+﻿/*
+ *  Copyright (c) 2014-2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant 
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.Owin.StaticFiles;
+
+namespace React.Owin
+{
+    /// <summary>
+    /// Enables serving static JSX files transformed to pure JavaScript. Wraps around StaticFileMiddleware.
+    /// </summary>
+    public class JsxFileMiddleware
+    {
+        private readonly StaticFileMiddleware _internalStaticMiddleware;
+
+        static JsxFileMiddleware()
+        {
+            Initializer.Initialize(_ => _);
```

Comment:
React.Web and React.MsBuild call the Initialize method internally, so I thought that it would be good idea to call it in React.Owin when the first Middleware object is created. 

If you're asking for the lambda, then I didn't know what to put there. I see that in https://github.com/reactjs/React.NET/blob/master/src/React.MSBuild/MSBuildHost.cs#L41  there's a call to "AsSingleton" method. Do you think I should use it here also?